### ISSUE TITLE
Preference Sync fails to sign in on MacOS

### ISSUE THREAD

**Issue by jaredready** [microsoft/vscode#95701]
- VSCode Version: 1.45.0-insider (commit 92ff20ac39c7507fdf4164e2bedfdfb700805fe1)
- OS Version: MacOS 10.14.6 (18G3020)

Steps to Reproduce:

1. Enable Preference Sync
2. Successfully follow authentication process

Does this issue occur when all extensions are disabled?: **Yes**/No

**Error message:**

Writing login information to the keychain failed with error 'The specified item could not be found in the keychain.'.

Source: Microsoft Account (Extension)

**Comment by RMacfarlane** [comment]
@jaredready Can you try using the most recent insider's version?

**Comment by jaredready** [comment]
I tested it on commit ec0d19789f89d4c25895f4a0cf364276cfb0c6ed and experienced the same issue

**Comment by RMacfarlane** [comment]
What's in the output panel for "Account"?

**Comment by jaredready** [comment]
Sure. Here it is:

```
[Info  - 14:25:09.236] Logging in...
[Info  - 14:25:12.664] Exchanging login code for token
[Info  - 14:25:13.16] Exchanging login code for token success
[Info  - 14:25:13.17] Login successful
[Info  - 14:25:13.21] Token available from cache, expires in 3598995 milliseconds
[Info  - 14:25:13.21] Token available from cache, expires in 3598995 milliseconds
[Error  - 14:25:13.25] Setting token failed: Error: The specified item could not be found in the keychain.
[Info  - 14:25:32.125] No stored keychain data, clearing local data
```

**Comment by RMacfarlane** [comment]
@jaredready Did you have multiple windows open when doing this?
